furusiya/view/data/saveload.py

```diff
@@ -0,0 +1,28 @@
+import shelve
+
+
+class SaveLoad:
+    def __init__(self, game):
+        self.game = game
+
+    def save(self):
```

Comment:
This needs tests. Maybe something as simple as calling `load(save(x))` and verify that all the properties are still set. We'll need this safety net later when we add stuff and have to worry about how it'll work with existing saves.